### ISSUE TITLE
Bug: Debug bar javascript issue with external script toggle

### ISSUE THREAD

**Issue by mihail-minkov** [codeigniter4/CodeIgniter4#5541]
### PHP Version

7.4

### CodeIgniter4 Version

4.1.6

### CodeIgniter4 Installation Method

Composer (using `codeigniter4/appstarter`)

### Which operating systems have you tested for this bug?

Windows

### Which server did you use?

apache

### What happened?

I am using Foundation 6.7 for front-end and CodeIgniter 4.1.6 for the back-end.

I thought it was an issue with a library inside Foundation till I revised what the code did. 

I basically get this error when toggling my element:

```
Uncaught TypeError: Cannot read properties of null (reading 'getAttribute')
    at HTMLAnchorElement.toggleRows (<anonymous>:145:45)
```

That is inside the ciDebugBar element in the toggleRows function:

![image](https://user-images.githubusercontent.com/4673722/148130706-4727fc84-fbe1-4456-a9cc-e752f58cd894.png)

### Steps to Reproduce

I use the off-canvas element of Foundation and a toggle button which uses the `data-toggle` button.

You can check it here https://get.foundation/sites/docs/off-canvas.html 

The smallest reproducible code would be:

```html
<!doctype html>
<html>
<head>
 <meta charset="utf-8">
 <script src="https://cdnjs.cloudflare.com/ajax/libs/foundation/6.7.4/js/foundation.min.js" integrity="sha512-+CnEgueYRDzx2s3IIAOpZd8HeOfrSbK3loeHw0jXJODAe3eFROHpBxEsun2kU+sZCaoAA0eFJrhfVV3RGG50GA==" crossorigin="anonymous" referrerpolicy="no-referrer"></script>
 <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/foundation/6.7.4/css/foundation.min.css" integrity="sha512-TgmH0v8FUwmsr3yDgd5PTCgR6lRZ2Q5c7KsUNTHcoxZpOExCX16MYECIL4xdRQOhQlz7pCnZlmA4zda58QWxBw==" crossorigin="anonymous" referrerpolicy="no-referrer" />
</head>
<body>
<div class="off-canvas position-left" id="menu-principal" data-off-canvas></div>
<div class="off-canvas-content" data-off-canvas-content>
    <main id="main-content">        
        <a data-toggle="menu-principal">Ver menu</a>
    </main>
</div>
</body>
</html>
```

### Expected Output

I don't expect an error inside console.

**Comment by sfadschm** [comment]
Ref. #5334 

I believe, that this part here should rather hook the `toggleRows` function to all `data-toggle` elements in the CI toolbar and not the whole `document`:

https://github.com/codeigniter4/CodeIgniter4/blob/ccc4a5e8f45072de35b07da9e639f6f120880532/system/Debug/Toolbar/Views/toolbar.js#L62-L67

Summoning @lonnieezell, who introduced this.

**Comment by sfadschm** [comment]
Should be closed by #5544

**Comment by kenjis** [comment]
Probably nobody confirmed the fix.
The current `develop` has broken Debug Toolbar.

**Comment by kenjis** [comment]
@mihail-minkov Thank you for reporting.
This bug should have been fixed in the latest `develop` branch.
Could you check?

**Comment by mihail-minkov** [comment]
> @mihail-minkov Thank you for reporting. This bug should have been fixed in the latest `develop` branch. Could you check?

No problem, anytime! So, I suppose the fix would be rolled out with 4.1.7 right? @kenjis

**Comment by kenjis** [comment]
@mihail-minkov Yes, the fix will be included in v4.1.7.
